### ISSUE TITLE
The reproduction results are inconsistent with the findings reported in the original paper

### ISSUE THREAD

**Issue by smailAA** [rllm-team/rllm#220]
My reproduction results are inconsistent with the results in the paper.

results of tml1m:
![Image](https://github.com/user-attachments/assets/dbcdae3e-542f-49d1-b6d8-144e755d6a9a)

results of tlf2k:
![Image](https://github.com/user-attachments/assets/b1ad70e1-d27e-4100-967e-44dac07a634f)

I want to know why the results improved? how did you improve it?
Also, why there is "BEST TEST ACC"? Is it used in "early-stop"?

**Comment by JianwuZheng413** [comment]
Thank you for your issue.

1. The improvement in the result is due to the fact that we fixed certain errors in these three datasets. For example, in the tacm12k dataset, we wrongly marked some columns whose column types were originally Categorical as Numerical, and discovered and fixed this error in the subsequent work. The same is true of the other two datasets. It will be updated in the paper later.
2. "BEST TEST ACC" refers to the result of the test set when the BEST result "BEST VAL ACC" is obtained on the validation set. In the actual training process, the validation set is used to select the model that performs best on it, evaluate the generalization ability of the model on unseen data, and at the same time avoid the evaluation deviation caused by the information leakage of the test set. The related “early-stopping” mechanism is also based on the validation set: training is prematurely terminated when the results of the validation set no longer improve within several rounds to prevent the model from overfitting on the training set.

**Comment by JianwuZheng413** [comment]
Now we have fixed this issue that might lead to misunderstandings in #223 .
